### PR TITLE
[Feature] Remove unnecessary findUserRole

### LINKED ISSUE #1882: [Feature] Reduce unnecessary database join
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/streampark/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Reduce unnecessary database join

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by 1996fanrui**
## What changes were proposed in this pull request

Issue Number: close #1882 

Remove unnecessary findUserRole

## Brief change log

Remove unnecessary findUserRole

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): no

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/resources/mapper/system/RoleMapper.xml [hunk 1]
```diff
@@ -32,8 +32,7 @@
         select r.*
         from t_role r
         left join t_member m on r.role_id = m.role_id
-        left join t_user u on u.user_id = m.user_id
-        where u.user_id = #{userId}
+        where m.user_id = #{userId}
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/resources/mapper/system/RoleMapper.xml]: query condition `team_id` is missing, please check.

Comment (1996fanrui) [reply streampark-console/streampark-console-service/src/main/resources/mapper/system/RoleMapper.xml]: After check, I don't know why need to call `findUserRole`?

There are two places in the code where roles are used, but I don't know why?

1. `ShiroRealm.doGetAuthorizationInfo` -> `simpleAuthorizationInfo.setRoles(roleSet)`
2. `UserServiceImpl.generateFrontendUserInfo` -> `userInfo.put("roles", roles);` 

Do you know why need roles here? As I understand, front-end just needs the `permissions` and `routers`.

Comment (wolfboys) [reply streampark-console/streampark-console-service/src/main/resources/mapper/system/RoleMapper.xml]: > After check, I don't know why need to call `findUserRole`?
> 
> There are two places in the code where roles are used, but I don't know why?
> 
> 1. `ShiroRealm.doGetAuthorizationInfo` -> `simpleAuthorizationInfo.setRoles(roleSet)`
> 2. `UserServiceImpl.generateFrontendUserInfo` -> `userInfo.put("roles", roles);`
> 
> Do you know why need roles here? As I understand, front-end just needs the `permissions` and `routers`.

hi 1996fanrui:

Currently, the `roles` is not actually used,  The front-end has a role-related instruction: `hasRole`, which is used for permission control, but it is not currently used.  Shouldn't use it in the future, so, The method and called references can be deleted.

Comment (1996fanrui) [reply streampark-console/streampark-console-service/src/main/resources/mapper/system/RoleMapper.xml]: @wolfboys  Thanks for your clarification, updated.

Comment (wolfboys) [reply streampark-console/streampark-console-service/src/main/resources/mapper/system/RoleMapper.xml]: > @wolfboys Thanks for your clarification, updated.

👌I'll review it later

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**